ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-main.yaml

```
@@ -86,6 +90,17 @@ tests:
       IMAGE_TAG: latest
       REGISTRY_ORG: open-cluster-management
     workflow: ocm-ci-image-mirror
+- as: golangci-lint
+  commands: |
+    export GOCACHE=/tmp/
+    export GOLANGCI_LINT_CACHE=/tmp/.cache
+    command -v go
+    go version
+    command -v golangci-lint
+    golangci-lint version
+    make golangci-lint
```

The build machinery assumes that the shell is Bash:

```suggestion
    make golangci-lint SHELL=/bin/bash
```